qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/settings/capabilities/ServerCapabilitiesInitializer.java

```diff
@@ -46,7 +47,8 @@ public static ServerCapabilities getNonDynamicServerCapabilities(ClientCapabilit
 		serverCapabilities
 				.setDocumentSymbolProvider(!clientCapabilities.isDocumentSymbolDynamicRegistrationSupported());
 		serverCapabilities.setHoverProvider(!clientCapabilities.isHoverDynamicRegistered());
-		serverCapabilities.setCodeActionProvider(!clientCapabilities.isCodeActionDynamicRegistered());
+		if (!clientCapabilities.isCodeActionDynamicRegistered())
+			serverCapabilities.setCodeActionProvider(DEFAULT_CODE_ACTION_OPTIONS);

```

Comment:
You need too to set DEFAULT_CODE_ACTION_OPTIONS in https://github.com/redhat-developer/quarkus-ls/blob/8bd86c7d1bb1aef8948803ee24754e3eb1dd7006/qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/settings/capabilities/QuteCapabilityManager.java#L83